src/harness/cbsd_sas_interface.py

```diff
@@ -0,0 +1,167 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
```

Comment:
I suggest either:
1) do not rename this file (I prefer)
or
2) move SasAdminInterface to another file, because it is not only for cbsd-sas test.